bin/build_hipamd.sh

```diff
@@ -86,14 +86,20 @@ if [ "$1" != "nocmake" ] && [ "$1" != "install" ] ; then
   export ROCM_RPATH="$AOMP_ORIGIN_RPATH_LIST"
   MYCMAKEOPTS="$AOMP_ORIGIN_RPATH -DCMAKE_BUILD_TYPE=$BUILDTYPE \
  -DCMAKE_PREFIX_PATH=$AOMP_INSTALL_DIR \
- -DCMAKE_CXX_FLAGS=-I$AOMP/include/amd_comgr \
  -DCMAKE_INSTALL_PREFIX=$AOMP_INSTALL_DIR \
  -DCMAKE_INSTALL_LIBDIR=lib \
  -DHIP_COMMON_DIR=$HIP_DIR \
  -DAMD_OPENCL_PATH=$OPENCL_DIR \
  -DROCCLR_PATH=$ROCclr_DIR \
  -DROCM_PATH=$ROCM_PATH"
 
+	# Enable HIPAMD Sanitizer Build
+	if [ "$AOMP_BUILD_SANITIZER" == 'ON' ]; then
+		MYCMAKEOPTS+=" -DCMAKE_CXX_FLAGS=-I$SANITIZER_COMGR_INCLUDE_PATH ${SANITIZER_FLAGS} -DCMAKE_C_FLAGS=${SANITIZER_FLAGS}"
+	else
+		MYCMAKEOPTS+=" -DCMAKE_CXX_FLAGS=-I$AOMP_include/amd_comgr"
+	fi
+
```

Comment:
Indentation

Reply:
I messed up upstream fork and local merge commit history. I will push a new pull request for this after resolving all your comments.